engine/src/test/java/org/camunda/bpm/engine/test/api/repository/DeleteProcessDefinitionTest.java

```diff
@@ -56,6 +42,20 @@
 import org.junit.Rule;
 import org.junit.Test;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static junit.framework.TestCase.assertNotNull;
+import static junit.framework.TestCase.assertNull;
+import static junit.framework.TestCase.assertTrue;
+import static junit.framework.TestCase.fail;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.camunda.bpm.engine.test.api.repository.RedeploymentTest.DEPLOYMENT_NAME;
+import static org.junit.Assert.assertEquals;
```

Comment:
🔧 Ideally, we organize imports as described in our [style guide](https://github.com/camunda/camunda-bpm-platform/wiki/Coding-Style-Java#imports). In practice, we usually do static imports first in one block, and regular imports afterward in another block. Older classes sometimes don't adhere to this style yet. We then reformat them accordingly.